src/Alert.php

```diff
@@ -121,15 +170,128 @@ public function options(array $value): self
     }
 
     /**
-     * Renders the alert body and the close button (if any).
+     * Enable/Disable encode body
      *
-     * @throws JsonException
+     * @param bool $encode
      *
-     * @return string the rendering result
+     * @return self
+     */
+    public function encode(bool $encode = true): self
+    {
+        $new = clone $this;
+        $new->encode = $encode;
+
+        return $new;
+    }
+
+    /**
+     * Enable/Disable dissmiss animation
+     *
+     * @param bool $fade
+     *
+     * @return self
+     */
+    public function fade(bool $fade = true): self
+    {
+        $new = clone $this;
+        $new->fade = $fade;
+
+        return $new;
+    }
+
+    /**
+     * Set type of alert, 'alert-success', 'alert-danger' etc
+     *
+     * @param string $type
+     *
+     * @return self
      */
-    private function renderBodyEnd(): string
+    public function type(string $type): self
```

Comment:
Why do you need this method, if you have an individual one for each type, it seems to be redundant.

Reply:
I have custom bootstrap css, with some additional color classes for all prefixes (text-*, bg-*, btn-* etc)

Reply:
So rename it to `addCssClass()`, sounds better `type` doesn't exist in `bootstrap5`.